nixio/property.py

```diff
@@ -115,6 +115,15 @@ def _create_new(cls, nixparent, h5parent, name, dtype, oid=None):
 
         return newentity
 
+    def append_values(self, data):
+        """Method for appending values to existing values"""
```

Comment:
The docstring shouldn't say what the method does, not describe what it is.

E.g., "Appends values to existing data."